include/envoy/router/router.h

```diff
@@ -361,6 +361,11 @@ class ShadowPolicy {
    *         present.
    */
   virtual const envoy::type::v3::FractionalPercent& defaultValue() const PURE;
+
+  /**
+   * @return true if the trace span should be sampled.
+   */
+  virtual bool sampled() const PURE;
```

Comment:
Please rename all the various methods/fields to `traceSampled()` or similar to match the proto field.

Reply:
Updated and aligned the two fields.